spec/feature/authenticated_admin_spec.rb

```diff
@@ -220,39 +220,7 @@
       expect(page).to have_content("desc")
       expect(page).to have_content("$10.00")
     end
-    within(".hidden_items") do
-      expect(page).to_not have_content("Bacon")
-      expect(page).to_not have_content("desc")
-      expect(page).to_not have_content("$10.00")
-    end
-  end
-
-  xit "can reveal an item previously hidden from cat index page" do
-    category = Category.create(name: "Breakfast")
-    item = Item.create(title: "Bacon",
-                       description: "desc",
-                       price: 1000,
-                       status: "hidden")
-    item.categories << category
-    allow_any_instance_of(ApplicationController).to receive(:current_user).
-                                                 and_return(admin)
-    visit category_path(category)
-    within(".hidden_items") do
-      expect(page).to have_content("Bacon")
-      expect(page).to have_content("desc")
-      expect(page).to have_content("$10.00")
-      click_link_or_button "Reveal"
-    end
-    within(".shown_items") do
-      expect(page).to have_content("Bacon")
-      expect(page).to have_content("desc")
-      expect(page).to have_content("$10.00")
-    end
-    within(".hidden_items") do
-      expect(page).to_not have_content("Bacon")
-      expect(page).to_not have_content("desc")
-      expect(page).to_not have_content("$10.00")
-    end
```

Comment:
Much cleaner logic.  I think this test is much more readable and prices.

_Yesterday it worked. 
Today it is not working. 
Rails is like that._